fastplotlib/layouts/_gridplot.py

```diff
@@ -281,3 +288,61 @@ def __next__(self) -> Subplot:
 
     def __repr__(self):
         return f"fastplotlib.{self.__class__.__name__} @ {hex(id(self))}\n"
+
+
+class GridPlotToolBar:
+    def __init__(self,
+                 plot: GridPlot):
+        """
+        Basic toolbar for a GridPlot instance.
+
+        Parameters
+        ----------
+        plot:
+        """
+        self.plot = plot
+
+        self._tools = list()
+
+        auto_tool = Button(value=False, disabled=False, icon='expand-arrows-alt', layout=Layout(width='auto'))
+        center_tool = Button(value=False, disabled=False, icon='compress-arrows-alt', layout=Layout(width='auto'))
+        panzoom_tool = ToggleButton(value=False, disabled=False, icon='hand-pointer', layout=Layout(width='auto'))
+        maintain_aspect_tool = ToggleButton(value=False, disabled=False, description="1:1", layout=Layout(width='auto'))
+        maintain_aspect_tool.style.font_weight = "bold"
+        self._tools.extend([auto_tool, center_tool, panzoom_tool, maintain_aspect_tool])
+
+        positions = [[i, j] for i, j in self.plot._get_iterator()]
+        values = list()
+        for pos in positions:
+            if self.plot[pos].name is not None:
+                values.append(self.plot[pos].name)
+            else:
+                values.append(tuple(pos))
+        self._dropdown = Dropdown(options=values, disabled=False, description='Plots:')
+
+        def auto_scale(obj):
+            current = self._dropdown.value
```

Comment:
Make a property that parses the Subplot and determines if its str or numerical. If it starts with "(" we can assume its numeric for now. This property returns the Subplot instance.